test/webServiceRequestHandlers.test.ts

```diff
@@ -0,0 +1,76 @@
+// TODO: rewrite all the outputprovider handle tests (old ones kept for reference)
```

Comment:
I implement this TODO in another PR. Should have probably created this file in that one, but I will send the other one in right after this one, so this shouldn't be a problem.